source/_docs/z-wave.markdown

```diff
@@ -3,6 +3,17 @@ title: "Z-Wave"
 description: "Using Z-Wave with Home Assistant."
 ---
 
+Home Assistant currently offers two Z-Wave integrations. 
+
+1. [Z-Wave](https://www.home-assistant.io/integrations/zwave/) based on OpenZWave 1.4 (EOL)
```

Comment:
Our integration is not EOL.

Reply:
Correct, however OpenZWave 1.4 is EOL. That is what it reads.

Reply:
That is not what it reads, the subject of this sentence is the Z-Wave integration of Home Assistant.

Reply:
I clarified the subject in a new commit, so that it reads that OpenZWave 1.4 = EOL and not the integration itself.